frappe/core/doctype/user/user.json

```diff
@@ -606,6 +612,45 @@
    "fieldtype": "Link",
    "label": "Module Profile",
    "options": "Module Profile"
+  },
+  {
+   "fieldname": "column_break_75",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "0",
+   "depends_on": "eval:(doc.document_follow_notify== 1)",
+   "fieldname": "follow_created_documents",
+   "fieldtype": "Check",
+   "label": "Automatically Follow Documents that you create"
+  },
+  {
+   "default": "0",
+   "depends_on": "eval:(doc.document_follow_notify== 1)",
+   "fieldname": "follow_commented_documents",
+   "fieldtype": "Check",
+   "label": "Automatically Follow Documents that you comment on"
+  },
+  {
+   "default": "0",
```

Comment:
defaults should be 1 (current behavior)

Reply:
We need to change the current behavior.
For a user, the number of documents followed reached 70k most of them were not needed to be followed
People who really want this can turn it on